### PR TITLE
Custom radio validation for equipment_type

### PR DESCRIPTION
**Pull request by alex-ju**
Use custom error message via `validate_radio_field ` for `equipment_type` field.

<table>
<tr><th>Before</th><th>After</th></tr>
<tr><td valign="top">

localhost_5000_product-details (1)

</td><td valign="top">

localhost_5000_product-details

[Trello card](https://trello.com/c/ckf2yaxx)

</td></tr>
</table>

### CODE REVIEW

**Review by bilbof — APPROVED** [review]
👍

File: app/controllers/coronavirus_form/product_details_controller.rb [hunk 1]
```diff
@@ -47,9 +47,10 @@ def selected_made_in_uk?
 
   def validate_fields(product)
     missing_fields = validate_missing_fields(product)
+    product_equipment_type = selected_ppe? ? validate_radio_field("#{PAGE}.equipment_type", radio: product["equipment_type"]) : []
```
Comment (alex-ju) [inline app/controllers/coronavirus_form/product_details_controller.rb:50]: I don't like this line – please suggest something nicer 😁

Comment (alex-ju) [reply app/controllers/coronavirus_form/product_details_controller.rb:50]: Or maybe something for later

Comment (bilbof) [reply app/controllers/coronavirus_form/product_details_controller.rb:50]: I think if it works lets ship it, I've not liked most of the PRs I've put up recently 😄I'm hoping we'll be given time for improving the codebase post-launch!

**Pull request merged**